src/types/instances.ts

```diff
@@ -0,0 +1,123 @@
+import { ethers } from 'ethers';
+
+import { CreateProposalParams, CreateZDAOParams } from './params';
+import { Choice, ProposalId, VoteId, zNA } from './primitives';
+import {
+  ProposalProperties,
+  TokenMetaData,
+  Transaction,
+  Vote,
+  zDAOAssets,
+  zDAOProperties,
+} from './structures';
+
+export interface SDKInstance {
+  /**
+   * Get all the list of zDAO
+   * @returns list of zNA
+   */
+  listZDAOs(): Promise<zNA[]>;
```

Comment:
Is this listing zNA's, zDAO's or zNA's that are associated with zDAO's?

The name of the function is confusing.